### DISCUSSION TITLE
Error message help please

### DISCUSSION THREAD

**Discussion by Splodge2001** [007revad/Synology_Plex_Backup#4] [General]
I am a total newbee at this, so please excuse any errors I have made in trying to run this script on my Synology DS218+.
I have created a backup directory on the NAS to match the one in the script.
I have copied the script to the NAS, but when I try to run it I get the following error:-

/volume1/Backups/scripts/backup_synology_plex_to_tar.sh: line 8: syntax error near unexpected token `newline'
/volume1/Backups/scripts/backup_synology_plex_to_tar.sh: line 8: `<!DOCTYPE html>'

I suspect that I am doing something wrong, but not having any knowledge I don't know how to fix it.
I would appreciate any help you could offer as having recently suffer eda HDD crash  I don't want to have to rebuild my Plex database again!

**Comment by 007revad** [comment]
It sounds like you may have edited the backup_synology_plex_to_tar.sh file in notepad on a PC or similar - which has changed some of the line endings in the file. I use [Notepad++](https://notepad-plus-plus.org/downloads/) in Windows because it keeps the original line endings.

Or maybe you didn't download the file correctly. Download the latest "source code.zip" and save it to your Synology, then extract the .zip file. https://github.com/007revad/Synology_Plex_Backup/releases

**Reply by Splodge2001** [reply]
That's it - sorted. 
Really grateful for your help. 
Running the first backup as I type this.
